File: abc.com/src/main/java/application/office/Office.java
package application.office;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import javax.validation.constraints.NotNull;

@Document
public class Office {

    @Id
    private String id;
    @NotNull
    private String address;
    @NotNull
    private int tel;
    private int fax;
    private String companyId;

    public Office() {

    }

    public  Office(String address, int tel, int fax, String companyId){
        this.address = address;
        this.tel = tel;
        this.fax = fax;
        this.companyId = companyId;
    }

    public String getId(){
        return this.id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getAddress() {
        return this.address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public int getTel() {
        return this.tel;
    }

    public void setTel(int tel) {
        this.tel = tel;
    }

    public int getFax() {
        return this.fax;
    }

    public String getCompanyId() {
		return companyId;
	}

	public void setCompanyId(String companyId) {
		this.companyId = companyId;
	}

	public void setFax(int fax) {
        this.fax = fax;
    }

  
}

File: abc.com/client/src/app/office/office.ts
import { Class } from '@angular/core';

export class Office {

  private  id: string;
  private address: string;
  private fax: string;
  private  telephone: number;
  private companyId:string;
  private  companyName: string;

  
  constructor() {
  }
  
  public setId ( id: string ) {
    this.id = id;
  }
  public getId (): string {
    return this.id ;
  }
  
  public setAddress ( address: string ) {
    this.address = address;
  }
  
  public getAddress (): string {
    return this.address ;
  }
  
   public setFax ( fax: string ) {
    this.fax = fax;
  }
  
  public getFax (): string {
    return this.fax ;
  }
  
   public setTelephone ( telephone: number ) {
    this.telephone = telephone;
  }
  
  public getTelephone (): number {
    return this.telephone ;
  }
  
   public setCompanyId( companyId: string ) {
    this.companyId = companyId;
  }
  
  public getCompanyId (): string {
    return this.companyId ;
  }
  public setCompanyName( companyName: string ) {
    this.companyName = companyName;
  }
  public getCompanyName (): string {
    return this.companyName ;
  }
     
}
